public/app/features/alerting/components/AlertDefinitionItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { css } from 'emotion';
+// @ts-ignore
+import Highlighter from 'react-highlight-words';
+import { Card, FeatureBadge, Icon } from '@grafana/ui';
+import { AlertDefinition } from 'app/types';
+import { FeatureState } from '@grafana/data';
+
+interface Props {
+  alertDefinition: AlertDefinition;
+  search: string;
+}
+
+export const AlertDefinitionItem: FC<Props> = ({ alertDefinition, search }) => {
+  return (
+    <li
+      className={css`
+        width: 100%;
+      `}
+    >
```

Comment:
this li element seems very unnecessary, we can just change the alert-rule-list to display flex or use VerticalGroup

Reply:
yeah, the ol/li setup looked weird to me.